src/hooks/useHandleChangeSearch.ts

```diff
@@ -18,41 +18,72 @@ let timeout: null | ReturnType<typeof setTimeout> = null;
 
 const useHandleChangeSearch = () => {
   const dispatch = useAppDispatch();
-  const getRepository = useAppSelector((state) => state.search.repository);
+  const baseRepository = useAppSelector((state) => state.search.baseRepository);
+  const newRepository = useAppSelector((state) => state.search.newRepository);
 
-  const searchByRevisionOrEmail = async (
-    repository: Repository['name'],
+  const warningText =
```

Comment:
Perhaps it could be moved to `src\resources\Strings.tsx`?

Reply:
Yes, true!